integration/src/test/kotlin/kotlinx/benchmark/integration/SuiteSourceGeneratorTest.kt

```diff
@@ -20,4 +20,25 @@ class SuiteSourceGeneratorTest : GradleTest() {
             }
         }
     }
+
+    @Test
+    fun runBenchmarks() {
+        project("kotlin-multiplatform", true).let { runner ->
+            runner.run(":nativeBenchmarkGenerate")
```

Comment:
Could you please also check the generated benchmark sources for `js` and `wasmJs` targets?
It seems like for those targets the `RootBenchmark_Descriptor.kt ` is not generated.